Ploidy fixes

stschiff opened the pull request:
A new feature in validate in version 1.4.0.0 checks whether Genotype_Ploidy from the Janno file is consistent with genotype data.

When running this on the community-archive, I noticed that many samples had indeed heterozygotes, even though being marked as haploid in the Janno. I have updated the respective packages. There could be some more cases, as I have only run the usual first 100 SNPs, but I think I should have caught most.

nevrome commented:
Brilliant! I love it when the validation helps to make the data better.

I also ran `validate` with `--fullGeno` on the entire dataset and did not find any more packages with this issue. You seemed to have caught all of them by looking at the first 100 SNPs.

Why did you also remove/set to `n/a` the `Capture_Type` column for some of the affected .janno files?

stschiff commented:
Thanks, good to know that `--fullGeno` also didn't report more.

Well the thing with the `Capture_Type` just came up: For modern samples we don't use Capture, so that should be set to `n/a`. I don't want to make this a general rule, though, as - in principle - one _could_ sequence a modern-day genome with Capture. It's just that it typically isn't done.

nevrome commented:
Good catch! Did you search for that systematically as well? Or just fixed it when it came up together with the ploidy issue?

stschiff commented:
Hmm, I think I did not, no. Not sure exactly now.

nevrome commented:
I think this would be the relevant query, right?

```bash
qjanno "SELECT Poseidon_ID,Capture_Type FROM d(.) WHERE Date_Type = 'modern' AND Capture_Type IS NOT NULL"
```

Which reminds me, that qjanno should add a column with the path to the .janno file a given sample is coming from. Or even the name of the package. The R package already has a feature like this. This would make it much more easy to determine which packages are affected here. I think I'll squeeze this into https://github.com/poseidon-framework/qjanno/pull/4.

stschiff commented:
OK, I think I've got them all.

stschiff commented:
So, to summarise, I've changed the Genotype_Ploidy where necessary. There is one issue with one of the 1000 Genomes samples, which has heterozygotes in the genotype data, even though it should be pseudo-haploid pulldown. We need to check whether that's an error that came from the AADR. 

I also fixed a lot of modern samples that had `Capture_Type` set to `OtherCapture`, which in most cases should be `n/a` (because there is no capture).

@AyGhal do you want to approve this quickly? Because our release of trident-1.4.0.0 and in particular xerxes-1.0.0.0 rely on this change, I'd appreciate quick feedback, otherwise I will take the risk and merge this in myself. No pressure.
